hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/response/volume/OMVolumeSetOwnerResponse.java

```diff
@@ -53,29 +54,35 @@ public OMVolumeSetOwnerResponse(String oldOwner,
     this.newOwnerVolumeArgs = newOwnerVolumeArgs;
   }
 
+  /**
+   * For when the request is not successful or it is a replay transaction.
+   * For a successful request, the other constructor should be used.
+   */
+  public OMVolumeSetOwnerResponse(@Nonnull OMResponse omResponse) {
+    super(omResponse);
+    Preconditions.assertTrue(!omResponse.getStatus().equals(
```

Comment:
We can have a method for this precheck in a base class, which can be used in all the response classes.

Reply:
Sure. Will update it.